plugins/chat/assets/stylesheets/common/chat-message.scss

```diff
@@ -97,6 +97,7 @@
     display: inline-block;
     color: var(--primary-medium);
     font-size: var(--font-down-2);
+    user-select: none;
```

Comment:
we have a mixin for this I think, but not sure if it's needed anymore

Reply:
a mixin for one property?

Reply:
```scss
@mixin user-select($mode) {
  -webkit-user-select: $mode;
  user-select: $mode;
}
```